manager/impl/src/main/java/org/jboss/forge/furnace/manager/impl/AddonManagerImpl.java

```diff
@@ -217,12 +217,15 @@ private AddonActionRequest createRequest(final AddonInfo requestedAddonInfo, fin
     */
    private void collectRequiredAddons(AddonInfo addonInfo, List<AddonInfo> addons)
```

Comment:
That won't work if you're planning to install an addon in ````~/.forge/addons```` and you already have their required dependencies in ````FORGE_HOME/addons````, for example